### DISCUSSION TITLE
Composer dependencies require a PHP version ">= 7.3.4"

### DISCUSSION THREAD

**Discussion by larsneo** [statamic/cms#4561] [Troubleshooting]
when trying to install a module within the statamic-frontend (3.2.17) an error occures

Fatal error: Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 7.3.4". You are running 7.2.34-nmm1. in /www/htdocs/w123456/statamic/vendor/composer/platform_check.php on line 24

but the cli-version via php -v in the shell is already

PHP 7.4.23 (cli) (built: Sep  8 2021 04:43:38) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with the ionCube PHP Loader + ionCube24 v10.4.0, Copyright (c) 2002-2020, by ionCube Ltd.

installation of modules via command line works as expected. any clue about this?

**Comment by edalzell** [comment]
The version served by the web server can be different than the CLI version.

Is this locally? What OS are you on?

**Reply by larsneo** [reply]
the website is hosted by the german provider all-inkl. within their adminstration one can choose between different php versions for the web and the phpinfo() shows correctly 7.4.23. for ssh one has to set the php version manually and as mentioned php -v reports 7.4.23 in the shell. i've really no clue why platform_check.php reports 7.2.x (and even worse: updates within the frontend worked a couple of time ago)

**Reply by jackmcdade** [reply]
Are you able to open a support request with the host? Not sure what we can do more to help you here!

**Comment by jasonvarga** [comment]
If you're dealing with PHP issues anyway, maybe you should considering just upgrading to PHP 8.
